app/constructs/rds.ts

```diff
@@ -94,6 +95,6 @@ export class RDSConstruct extends Construct {
     endpoint: string,
     password: string
   ): string {
-    return `postgresql+pygresql://${dbConfig.masterUsername}:${password}@${endpoint}:${dbConfig.port}/${dbConfig.dbName}`;
+    return `postgresql+psycopg2://${dbConfig.masterUsername}:${password}@${endpoint}:${dbConfig.port}/${dbConfig.dbName}`;
```

Comment:
What's the motive behind this change? Are there any added benefits, as this is used only for metadata?

Reply:
psycopg is much more active open source project than pygresql https://github.com/psycopg https://github.com/PyGreSQL so that is why the change was made